sockeye/model.py

```diff
@@ -166,6 +169,10 @@ def load_params_from_file(self, fname: str):
                                                      "This is either not a model directory or the first training "
                                                      "checkpoint has not happened yet." % fname)
         self.params, self.aux_params = utils.load_params(fname)
+        utils.check_condition(all(name.startswith(self.prefix) for name in self.params.keys()),
```

Comment:
is it important to check this? What do we want to guard against?

Reply:
At this point not really, but I want to guard against future problems. Right now, the model prefix is hardcoded/defaults to an empty string (See SockeyeModel constructor), but if we would create multiple SockeyeModels in one graph in the future and name them differently, the parameters would be tied to that name. This is not necessarily what we want so maybe we would want to strip the prefix when loading the parameters in the future, but in order to not forget this, I am putting in this check now.

Reply:
Nit: This should be an assert. (I can be boring, I know ;-) )